src/features/search/components/SearchBox.tsx

```diff
@@ -32,13 +34,20 @@ import { SearchMainInput } from './SearchMainInput'
 
 const inset = 25 // arbitrary hitSlop zone inset for touchable
 const hitSlop: Insets = { top: inset, right: inset, bottom: inset, left: inset }
+const SEARCH_DEBOUNCE_MS = 500
 
-type Props = {
+type Props = UseSearchBoxProps & {
   searchInputID: string
   accessibleHiddenTitle?: string
+  appEnableAutocomplete: boolean
 }
 
-export const SearchBox = function SearchBox({ searchInputID, accessibleHiddenTitle }: Props) {
+export const SearchBox: React.FC<Props> = ({
```

Comment:
il me semble que `FC` est déprécié car pas super explicite

```suggestion
export const SearchBox: FunctionComponent<Props> = ({
```